[2025-03-03T01:51:06.422+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-03T01:51:06.442+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Stock_Alpha_Vantage.load manual__2025-03-03T01:50:58.704129+00:00 [queued]>
[2025-03-03T01:51:06.448+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Stock_Alpha_Vantage.load manual__2025-03-03T01:50:58.704129+00:00 [queued]>
[2025-03-03T01:51:06.448+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-03T01:51:06.458+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2025-03-03 01:50:58.704129+00:00
[2025-03-03T01:51:06.465+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Stock_Alpha_Vantage', 'load', 'manual__2025-03-03T01:50:58.704129+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/Assignment5.py', '--cfg-path', '/tmp/tmp4sqa1zj7']
[2025-03-03T01:51:06.469+0000] {standard_task_runner.py:105} INFO - Job 95: Subtask load
[2025-03-03T01:51:06.468+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5009) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-03T01:51:06.470+0000] {standard_task_runner.py:72} INFO - Started process 5014 to run task
[2025-03-03T01:51:06.521+0000] {task_command.py:467} INFO - Running <TaskInstance: Stock_Alpha_Vantage.load manual__2025-03-03T01:50:58.704129+00:00 [running]> on host e39575d078f8
[2025-03-03T01:51:06.617+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Stock_Alpha_Vantage' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-03-03T01:50:58.704129+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-03T01:50:58.704129+00:00'
[2025-03-03T01:51:06.619+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-03T01:51:06.635+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-03T01:51:06.636+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-03T01:51:06.636+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-03T01:51:07.706+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-03T01:51:07.877+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-03T01:51:07.972+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-03T01:51:08.178+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-03T01:51:08.180+0000] {logging_mixin.py:190} INFO - Table 'raw.stock_data' created or already exists.
[2025-03-03T01:51:08.315+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-03T01:51:08.318+0000] {logging_mixin.py:190} INFO - (datetime.datetime(2025, 2, 25, 14, 24, 16, 382000, tzinfo=<DstTzInfo 'America/Los_Angeles' PST-1 day, 16:00:00 STD>), 'STOCK_DATA', 'MYDATABASE', 'RAW', 'TABLE', '', '', 180, 7680, 'ACCOUNTADMIN', '1', 'OFF', 'OFF', 'OFF', None, None, 'N', 'N', 'ROLE', 'N', None, 'N', 'N', 'N', 'N')
[2025-03-03T01:52:20.313+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-03T01:52:20.317+0000] {logging_mixin.py:190} INFO - Data inserted successfully!
[2025-03-03T01:52:20.318+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-03T01:52:20.346+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-03T01:52:20.347+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Stock_Alpha_Vantage, task_id=load, run_id=manual__2025-03-03T01:50:58.704129+00:00, execution_date=20250303T015058, start_date=20250303T015106, end_date=20250303T015220
[2025-03-03T01:52:20.445+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-03T01:52:20.493+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-03T01:52:20.495+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
